pydra/engine/helpers.py

```diff
@@ -146,9 +146,7 @@ def execute(cmd):
         loop = asyncio.ProactorEventLoop()  # for subprocess' pipes on Windows
```

Comment:
are we actually support windows? anyone tested it?

Reply:
we haven't, but pydra should try to be cross-platform